resources/backup_scripts/code-backup.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
```

Comment:
I read through the script, it looks OK. To be honest, a python script to wrap around `aws s3 sync /path/to/files s3://path/to/bucket` may not be entirely necessary, but since it's written we can make use of it. Could you please add an example either in the header of this file, or in a README in the `backup_scripts` directory with a sample cron job configuration for this script?